### PR TITLE
add a null page view when all windows lose focus

### PR DESCRIPTION
**Pull request by bridiver**
cc @mrose17

### CODE REVIEW

**Comment by bbondy** [comment]
Could you link up the issue? Thanks.

File: js/stores/eventStore.js [hunk 1]
```diff
@@ -101,6 +108,16 @@ const doAction = (action) => {
       addPageView(action.frameProps.get('location'))
       lastActiveTabId = action.frameProps.get('tabId')
       break
+    case AppConstants.APP_WINDOW_BLURRED:
+      windowBlurred(action.appWindowId)
+      break
+    case AppConstants.APP_IDLE_STATE_CHANGED:
+      if (action.idleState !== 'active') {
+        addPageView(null)
```
Comment (bsclifton) [inline js/stores/eventStore.js]: How does a null page view behave?

**Comment by bridiver** [comment]
need to fix the commit, it should say fixes #3650

File: js/stores/eventStore.js (hunk 1 above)
Comment (bridiver) [inline js/stores/eventStore.js]: it just logs the timestamp when the browser became inactive. We use the same thing for closing all windows

**Comment by bbondy** [comment]
Moving to 0.12.1 for now, please merge when reviewed @mrose17 it could make it in a respin still.

**Comment by mrose17** [comment]
@bbondy - i have reviewed this PR and it works well on my system.

@bridiver - we may need to iterate a bit in the future, specifically to deal with mouse enter events and scrolling events; however, i'd like us to drive this version for a while before digging into it again.

many thanks for the fast updates!

**Pull request merged**